docs/guides/configuration-guide/proxy.md

```diff
@@ -23,6 +23,16 @@ osism apply squid
 
 ## Configurations
 
+:::info
+It is advisable to exclude networks that are locally accessible from using the HTTP proxy
+if they use HTTP(S) communication, as otherwise communication will take place unnecessarily via the
+proxy. In some cases the proxy does not have access to the internal networks (this depends on its location),
+but this can also lead to higher latencies or to inferred availability problems
+if the proxy is temporarily unavailable.
+
+It usually makes sense to exclude all [private ipv4 networks](https://www.rfc-editor.org/rfc/rfc1918).
```

Comment:
Remove this. We have a lot of scenarios where this makes no sense.

Reply:
Interesting. Can you provide more details?